backend/src/federation/validateCommunities.ts

```diff
@@ -75,6 +81,36 @@ export async function validateCommunities(): Promise<void> {
   }
 }
 
+export async function writeJwtKeyPairInHomeCommunity(): Promise<DbCommunity> {
+  logger.debug(`Federation: writeJwtKeyPairInHomeCommunity`)
+  try {
+    // check for existing homeCommunity entry
+    let homeCom = await DbCommunity.findOne({ where: { foreign: false } })
```

Comment:
Maybe you can use getHomeCommunity from database here?
Or move the logic altogether into getHomeCommunity ?
So we can make sure everyone needing home community get's a valid home community with all keys?
Or into shared and call inside getHomeCommunity query?
Shared for all stuff, don't needing database and createKeyPair seems only to need jose.
It is a kind of migration, isn't? Then normally homeCom should get this keys on first initalization.